src/main/resources/config.json

```diff
@@ -1,6 +1,6 @@
 {
 
-   "channels": ["channelName"],
-   "token": "token"
+   "channels": ["sofie"],
+   "token": "NzUxOTMwMzc0NTk4MTY0NTkx.X1QPuw.MlF4iuwZCKOfQny4iFJwgSeKBpE"
```

Comment:
You should have kept the 
"channels": ["channelName"],
"token": "token"
instead of 
 "channels": ["sofie"],
   "token": "NzUxOTMwMzc0NTk4MTY0NTkx.X1QPuw.MlF4iuwZCKOfQny4iFJwgSeKBpE"